schema/compose-spec.json

```diff
@@ -103,12 +108,12 @@
                 "context": {"type": "string"},
                 "dockerfile": {"type": "string"},
                 "dockerfile_inline": {"type": "string"},
-                "entitlements": {"type": "array", "items": {"type": "string"}},
+                "entitlements": {"$ref": "#/definitions/list_of_strings"},
```

Comment:
Some of these could possibly be `list_of_unique_strings` but that would technically be a breaking change.